app/views/dashboards/_dashboard_article.html.erb

```diff
@@ -64,9 +64,8 @@
     <% elsif @user == article.user %>
       <a href="<%= article.path %>/delete_confirm" data-no-instant class="crayons-btn crayons-btn--danger">Delete</a>
     <% end %>
-    <% if @current_user_pro %>
-      <a href="<%= article.path %>/stats" data-no-instant class="crayons-btn">Stats</a>
-    <% end %>
+
+    <a href="<%= article.path %>/stats" data-no-instant class="crayons-btn">Stats</a>
```

Comment:
I wonder if we should add this to our URL helper module, e.g. 

```ruby
# app/lib/url.rb
def self.article_stats(article)
  URI.join(url(article.path), "stats")
end

# here
<a href="<%= URL.article_stats(article) %>" data-no-instant class="crayons-btn">Stats</a>
```

It's not super important, but it would be a bit more in line with how we deal with similar URLs elsewhere. This can of course be done in a separate PR.

Reply:
I might do this as a 15 minute follow up fix! I'm having really frustrating issues with Travis today and I kind of want to hold off on triggering another build here for fear of Travis just not picking it up. Plus, as you said, this could be a great follow up PR 👍🏽